net/openappid/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openappid
+PKG_VERSION:=26425
+PKG_RELEASE:=1
+
+PKG_SOURCE:=snort-$(PKG_NAME).tar.gz
+PKG_SOURCE_URL:=https://www.snort.org/downloads/openappid/
+PKG_HASH:=8513877ce2264bb22119d911c2cf11f73735c866e2ca0d061c35eef6740d51f9
+
+PKG_MAINTAINER:=John Audia <thereadgraysky@proton.me>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openappid
+  SUBMENU:=Firewall
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+snort3
+  TITLE:=Detection language and processing module for Snort
+  URL:=http://www.snort.org/
+  MENU:=1
```

Comment:
There isn't anything for menu, right? You can check it in make menuconfig, that it is empty. So, you can drop it.

Reply:
OK... yes, only a binary in the menuconfig.